src/dumpsxiso/main.cpp

```diff
@@ -964,7 +1068,8 @@ int Main(int argc, char *argv[])
 		"  -s <path>  - Outputs an MKPSXISO compatible XML script for later rebuilding.\n"
 		"  -S|--sort-by-dir - Outputs a \"pretty\" XML script where entries are grouped in directories, instead of strictly following their original order on the disc.\n"
 		"  -e|--encode <codec> - Codec to encode CDDA/DA audio. wave is default. Supported codecs: " SUPPORTED_CODEC_TEXT "\n"
-		"  -h|--help  - Show this help text\n";
+		"  -h|--help  - Show this help text\n"
+		"  -pt|--pathTable - instead of going throught the file system, go to every known directory in order; helps with deobfuscating\n";
```

Comment:
```suggestion
		"  -pt|--path-table - instead of going through the file system, go to every known directory in order; helps with deobfuscating\n";
```

`sort-by-dir` sets a precedent to use `this-style-of-naming` over `thisStyle`. Let's be consistent.